README.Rmd

```diff
@@ -87,6 +88,19 @@ Finally, you can calculate perceptual distances to a reference colour, as in
 distance(c("red","green","blue"), "red")
 ```
 
+### Minimal ggplot2 example
+```r
+library(magrittr)
+library(ggplot2)
+library(shades)
+data("mtcars")
+mtcars$cyl<- factor(mtcars$cyl)
+ggplot(mtcars, aes(mpg, qsec, col=cyl)) + geom_point() + 
+            scale_color_manual(values=gradient("viridis",3) %>% dichromat)
+ggplot(mtcars, aes(cyl, mpg, fill=cyl)) + geom_boxplot() + 
+            scale_fill_manual(values=gradient("viridis",3) %>% dichromat)
```

Comment:
These shouldn't go through `dichromat()`—that is only for simulating colour blindness, but the scales carry more information for those with normal colour perception in their original form